src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar
+
+data class Cars(private val cars: List<Car>) {
+    var positions = Positions()
```

Comment:
프로퍼티의 getter 함수를 노출하면서 setter 함수에 대한 접근을 제한하려면 setter 함수를 private으로 지정하면 됩니다.

```suggestion
    var positions = Positions()
        private set
```

Reply:
제가 놓쳤네요. 👍
e572b95684c3cede7ff21b169359810b55cbaf9d 에서 적용하였습니다.